shavar/e2e-test/prefs.ini

```diff
@@ -9,20 +9,28 @@ browser.safebrowsing.phishing.enabled = false
 browser.safebrowsing.malware.enabled = false
 privacy.trackingprotection.enabled = true
 browser.safebrowsing.provider.mozilla.nextupdatetime = 1
-browser.safebrowsing.provider.mozilla.gethashURL = https://shavar.stage.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.gethashURL = http://127.0.0.1
```

Comment:
My understanding is that the environments listed below do not refer to this gethashURL, so there was no need for it to be the stage environment.